### PR TITLE
Docs: update COC

### LINKED ISSUE #31: create docker file and publish to gh container registry

### LINKED ISSUE #82: [OTHER] Update COC
### What would you like to share?

New version: https://www.contributor-covenant.org/version/2/1/code_of_conduct/

### Additional information

_No response_

### Would you like to work on this issue?

- [ ] Yes, I want to work on this issue!

### PR DESCRIPTION
**Pull request by itsadityagupta**
## Fixes Issue

Closes #82 

## Changes proposed

- [x] - Updated COC

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/86"></a>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -50,21 +44,20 @@ decisions when appropriate.
...
+official social media account, or acting as an appointed representative at an online 
+or offline event.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-eddie@jaoudestudios.com.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported 
+to the community leaders responsible for enforcement at community@eddiehub.org.
```
Comment (suitupstan) [inline CODE_OF_CONDUCT.md]:
```suggestion
to the community leaders responsible for enforcement at eddie@jaoudestudios.com. 
```

**Review by suitupstan — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Comment by EmmaDawsonDev** [comment]
@Aditya-Gupta1 there is a small conflict created from another pr being merged into the project. Let us know when you’ve fixed it or if you need any help resolving the conflict.

**Comment by itsadityagupta** [comment]
Hi @emmalearnscode. I've fixed the conflicts.

**Review by suitupstan — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Review by EmmaDawsonDev — APPROVED** [review]
LGTM!

**Comment by EmmaDawsonDev** [comment]
Thanks for contributing @Aditya-Gupta1 !

**Pull request merged**